numpy/core/src/_simd/_simd_inc_convert.h

```diff
@@ -41,39 +41,38 @@ simd_scalar_to_obj(simd_data data, simd_data_type dtype)
     return PyLong_FromUnsignedLongLong(data.u64 >> leftb);
 }
 
+typedef struct {
+    size_t len;
```

Comment:
Thinking some more, this should probably be `Py_ssize_t`?

Reply:
it should be size_t or just unsigned int to avoid overflow since it should only accept unsigned integer.